sidecar/sidecar.go

```diff
@@ -284,7 +289,7 @@ func startProxy(conf *config.Config, discovery api.ServiceDiscovery, kubeClient
 	)
 	nginxProxy := proxy.NewNGINXProxy(nginxManager)
 
-	rules, err := buildServiceRules(conf, kubeClient)
```

Comment:
Can remove the `kubeClient` argument from `startProxy()`. It was there only for the kubernetes rules adapter.